FEATURE_PROCESS.adoc

```diff
@@ -1,5 +1,124 @@
+* [[planning_issue]]File an issue with the https://github.com/orgs/wildfly/projects/7[WildFly Feature Planning project]. To do this create an issue using the https://github.com/wildfly/wildfly-proposals/issues/new?assignees=&labels=feature&projects=wildfly%2F7&template=feature-development.yaml[Feature Development issue template]. This issue is important because:
+** It is what allows effective coordination of overall feature development for a WildFly release from a single, publicly visible, point.
+** It allows users to see an overall roadmap for WildFly features.
+* Draft a minimal feature analysis, based on the current https://github.com/wildfly/wildfly-proposals/blob/main/design-doc-template.adoc[design-doc-template]. This should include the expected stability level, links to the issues above, and enough Overview and User Story information to give people interested in being part of the feature team a sense of what would be involved.
+* Submit a draft analysis PR to the https://github.com/wildfly/wildfly-proposals[`wildfly-proposals`] repository.
+* Send an email to the mailto:wildfly-dev@lists.jboss.org[`wildfly-dev@lists.jboss.org`] mail list telling people about the upcoming work and asking for volunteers to participate in the feature team. (If you don't have a wildfly-dev list subscription, visit the https://lists.jboss.org/admin/lists/wildfly-dev.lists.jboss.org/[list subscription page] to create one. Posting is restricted to subscribers.)
+
+When the above tasks are complete, change the `Status` field of the WildFly Feature Planning project JIRA to `Planning`.
+
+=== Planning
+
+The `Planning` stage of a feature consists of <<feature-team,feature team>> formation and enough discussion of the feature among the team that they can agree on a target feature stability level and the target WildFly release in which they expect to complete the feature.
+
+NOTE: The target WildFly release is just a target and is not a commitment.
+
+Update the feature analysis document with the planned stability level (if it has changed) and the GitHub ids of the feature team members.
+
+If there is an available iteration value that matches the target WildFly release, set the `Iteration` field in the <<planning_issue,WildFly Feature Planning project issue>> to that value.
+
+NOTE: The expectation is there will always be iterations available for the next two WildFly releases. If teams expect to target a later release than that and there's no iteration available, that's ok; just don't set it.
+
+When the above tasks are complete, change the `Status` field of the WildFly Feature Planning project JIRA to `In Progress`.
+
+IMPORTANT: Changes to the `Iteration` field of a WildFly Feature Planning project issue should always reflect the consensus of the feature team.
+
+=== In Progress
+
+The main work on the feature happens in this stage. The feature team works to meet the <<requirements,requirements>> that apply to the feature's target stability.
```

Comment:
This is the kind of thing we can add later but as an open source project maybe we could make some suggestions about discussing publicly such as starting a Zulip thread or a thread on the dev list so others with an interest can follow the conversation.

Reply:
They are already asked to start a wildfly-dev thread. 

But adding text to encourage public discussion is ok. I am concerned though about this turning into a mega document that no one reads.

Reply:
Added.